Makefile

```diff
@@ -3,23 +3,11 @@
 SHELL = /bin/bash
 PROJECT_PATH := $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
 
-local-setup: ## Builds and deploys kourier locally in a "kind" cluster with knative, forwards the local 8080 to kourier/envoy
```

Comment:
Is anybody at this point even using `make` here? Should we just get rid of it?

Reply:
Yes, let's remove Makefile.